### PR TITLE
Bump docker to 9.4.0

### PR DESCRIPTION
**Pull request by onurctirtir**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+### citus-docker v9.4.0.docker (July 28, 2020) ###
+
+* Bump Citus version to 9.4.0
+
+* Bump PostgreSQL version to
```
Comment (onurctirtir) [inline CHANGELOG.md:5]:
```suggestion
* Bump PostgreSQL version to 12.3
```

File: Dockerfile-alpine [hunk 2]
```diff
@@ -1,5 +1,5 @@
 FROM postgres:12.2-alpine
```
Comment (hanefi) [inline Dockerfile-alpine:1]: Not 🆒

File: Dockerfile [hunk 3]
```diff
@@ -1,5 +1,5 @@
-FROM postgres:12.2
-ARG VERSION=9.3.5
+FROM postgres:12.3
```
Comment (hanefi) [inline Dockerfile:1]: :cool:

File: Dockerfile-alpine (hunk 2 above)
Comment (hanefi) [reply Dockerfile-alpine:1]:
```suggestion
FROM postgres:12.3-alpine
```

Comment (onurctirtir) [reply Dockerfile-alpine:1]: I should mention this in https://github.com/citusdata/packaging/issues/500

**Pull request merged**